backend/kesaseteli/kesaseteli/settings.py

```diff
@@ -248,9 +248,9 @@
 # Azure ADFS
 LOGIN_URL = "django_auth_adfs:login"
 
-ADFS_CLIENT_ID = env.str("ADFS_CLIENT_ID")
-ADFS_CLIENT_SECRET = env.str("ADFS_CLIENT_SECRET")
-ADFS_TENANT_ID = env.str("ADFS_TENANT_ID")
+ADFS_CLIENT_ID = env.str("ADFS_CLIENT_ID") or "client_id"
```

Comment:
I think env.str("ADFS_TENANT_ID", "client_id") works as well

Reply:
Seemed that `env.str("ADFS_TENANT_ID", "client_id")` did not work if this was set explicitly: `ADFS_TENANT_ID=`